frontend/public/components/modals/cluster-update-modal.tsx

```diff
@@ -42,10 +52,9 @@ class ClusterUpdateModal extends PromiseComponent {
     return <form onSubmit={this._submit} name="form" className="modal-content">
       <ModalTitle>Update Cluster</ModalTitle>
       <ModalBody>
-        <p>
-          For more detailed documentation on specific versions, &nbsp;
-          <ExternalLink href="https://github.com/openshift/origin/releases" text="view release notes." />
-        </p>
+        {/* <p>
```

Comment:
When do we need to resolve this TODO?

Reply:
Probably not for beta2. The link that we had here was just not useful, so I removed it until we have something real to put there.